packages/pwa-kit-create-app/assets/bootstrap/js/package.json.hbs

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.1",
   "license": "See license in LICENSE",
   "engines": {
-    "node": "^16.0.0 || ^18.0.0",
-    "npm": "^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0"
```

Comment:
Why do we define only Node 18 and Node 20 on a generated project, while we define Node 16, Node 18, and Node 20 in monorepo packages?

Reply:
I didn't want to break existing projects, if we have any customers who are still using node 16 for local development. (Which would be weird, but technically possible.) For newly generated projects, though, we don't have to worry about supporting legacy versions!

Reply:
Sounds good. I thought that was the case, in theory dropping Node 16 would be a breaking change.